llama_stack/providers/remote/inference/ollama/ollama.py

```diff
@@ -395,29 +396,25 @@ async def openai_completion(
             raise ValueError("Ollama does not support non-string prompts for completion")
 
         model_obj = await self._get_model(model)
-        params = {
-            k: v
-            for k, v in {
-                "model": model_obj.provider_resource_id,
-                "prompt": prompt,
-                "best_of": best_of,
-                "echo": echo,
-                "frequency_penalty": frequency_penalty,
-                "logit_bias": logit_bias,
-                "logprobs": logprobs,
-                "max_tokens": max_tokens,
-                "n": n,
-                "presence_penalty": presence_penalty,
-                "seed": seed,
-                "stop": stop,
-                "stream": stream,
-                "stream_options": stream_options,
-                "temperature": temperature,
-                "top_p": top_p,
-                "user": user,
-            }.items()
-            if v is not None
-        }
+        params = await prepare_openai_completion_params(
```

Comment:
Unrelated but why was `prepare_openai_completion_params` built as an async function? At a glance, it doesn't feel like it's needed :) Am I missing something?

Reply:
I don't think it's actually needed - it's probably just because it started life as the `convert_message_to_openai_dict*` type calls, morphed into its own method, and kept the async definition from that lineage vs convert it to synchronous def.